packages/manager/modules/pci/src/projects/project/storages/storage-users.service.js

```diff
@@ -41,6 +41,18 @@ export default class PciStoragesUsersService {
       .then(({ data }) => data);
   }
 
+  mapUsersToCredentials(projectId, users) {
+    const usersCredentialsPromises = users.map((user) =>
+      this.getS3Credentials(projectId, user.id).then((data) => {
+        const updatedUser = user;
+        [updatedUser.s3Credentials] = data;
+        return updatedUser;
+      }),
```

Comment:
**nitpick:**  Unnecessary local variable
```suggestion
      this.getS3Credentials(projectId, user.id).then((data) => ({
         ...user,
         s3Credentials: data,
      })),
```

Reply:
This re-writing works only if we want to add a new pair (key, value) to an existing object.
It doesn't work when we need to update the value of an object attribute.

Reply:
Done.